wagtailnhsukfrontend/mixins.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from wagtail.admin.edit_handlers import MultiFieldPanel, FieldPanel
+
+
+class ReviewDateMixin(models.Model):
+
+    last_review_label = models.CharField(verbose_name="Reviewed date label", max_length=120, help_text="The default for NHS frontend is: Page last reviewed:")
```

Comment:
Need to have a rethink about these labels. I don't think the label should be a field on every page. More likely users will want to define the label once, not have to repeat it on every new page they create.
(I'm not sure how to actually do this yet, need to have a think about it).